public/app/features/alerting/unified/components/notification-policies/ContactPointSelector.tsx

```diff
@@ -82,56 +65,13 @@ export const ContactPointSelector = ({
   }
 
   return (
-    <Stack>
-      <Select
-        virtualized={options.length > MAX_CONTACT_POINTS_RENDERED}
-        options={options}
-        value={matchedContactPoint}
-        {...selectProps}
-        isLoading={isLoading}
-        disabled={isLoading}
-      />
-      {showRefreshButton && (
-        <IconButton
-          name="sync"
-          onClick={onClickRefresh}
-          aria-label={t('alerting.contact-point-selector.aria-label-refresh-contact-points', 'Refresh contact points')}
-          tooltip={t(
-            'alerting.contact-point-selector.tooltip-refresh-contact-points-list',
-            'Refresh contact points list'
-          )}
-          className={cx(styles.refreshButton, {
-            [styles.loading]: loaderSpinning || isLoading,
-          })}
-        />
-      )}
-    </Stack>
+    <Select
```

Comment:
Can we also move this to Combobox for consistency with the other selector?

Reply:
The new Combobox should handle the virtualization for us

Reply:
I'll see if I can tackle this in a separate PR :D